File: app/src/main/java/com/antyzero/smoksmog/ui/screen/about/AboutActivityComponent.java
package com.antyzero.smoksmog.ui.screen.about;

import com.antyzero.smoksmog.ui.screen.ActivityModule;

import dagger.Subcomponent;

@Subcomponent(
        modules = {
                ActivityModule.class,
        }
)
public interface AboutActivityComponent {
    void inject( AboutActivity activity );
}

File: app/src/main/kotlin/com/antyzero/smoksmog/ui/screen/about/AboutActivityComponent.kt
package com.antyzero.smoksmog.ui.screen.about

import com.antyzero.smoksmog.ui.screen.ActivityModule

import dagger.Subcomponent

@Subcomponent(modules = arrayOf(ActivityModule::class))
interface AboutActivityComponent {
    fun inject(activity: AboutActivity)
}
